R/cpi_validate_raw.R

```diff
@@ -0,0 +1,95 @@
+#' Validate raw cpi data
+#'
+#' @param cpi raw cpi data, as loaded via `pipfun::load_from_gh`
+#' @param detail has an option TRUE/FALSE, default value is FALSE
+#' @import data.validator
+#' @importFrom assertr in_set not_na is_uniq
+#'
+#' @export
+cpi_validate_raw <- function(cpi, detail = getOption("pipaux.detail.raw")){
```

Comment:
I sent you an email showing you a real case of a problem that this check should find out... please add a check for that. THANKS!

Reply:
The case you have sent me via email shows same cpi value for multiple years. What would be the minimum years where the cpi value could be same? In the example, cpi value are same for four survey reference years. In the latest PR, I checked same cpi / duplicate cpi. 

Thanks